awscli/examples/rum/update-rum-metric-definition.rst

```diff
@@ -0,0 +1,11 @@
+**To modifies one existing metric definition for CloudWatch RUM extended metric**
+
+The following ``update-rum-metric-definition`` modifies one existing metric definition for CloudWatch RUM extended metrics. If the command succeeds, no output is returned. ::
+
+	aws rum update-rum-metric-definition \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general